graphein/protein/features/nodes/amino_acid.py

```diff
@@ -75,11 +73,17 @@ def meiler_embedding(n, d) -> pd.Series:
 
 
 def aaindex_1_feat(n, d, feature_name: str) -> pd.Series:
```

Comment:
This can go. Some prototyping I was doing. @cch1999 has contributed some proper functionality for AAindex.

Curious as to why we should minimise optional dependencies. If they're optional surely, compatibility drift is not a major cause for concern.

Reply:
That's true. OK, I guess we shouldn't worry too much about it. That said, though, for optional dependencies, raising an `ImportError` + some informative message on how to install the dependency might be a good idea. Here's an example:

```python
try:
    import some_package
except ImportError:
    raise ImportError("some_package is an optional dependency; to use functions in this submodule, please install some_package by using `pip install some_package` or `conda install -c conda-forge some_package`.")
```